_data/banking.yml

```diff
@@ -1,647 +1,653 @@
-      tfa: No
-
-    - name: Umpqua Bank
-      url: https://www.umpquabank.com/
-      twitter: askumpquabank
-      img:  umpqua.png
-      tfa: No
-
-    - name: Union Bank
-      url: https://www.unionbank.com/
-      twitter: UnionBank
-      img: unionbank.png
-      tfa: No
-
-    - name: US Bank
-      url: https://www.usbank.com/
-      twitter: usbank
-      img: usbank.png
-      tfa: No
-
-    - name: USAA
-      url: https://www.usaa.com/
-      img: usaa.png
-      tfa: Yes
-      sms: Yes
-      hardware: Yes
-      software: Yes
-      doc: https://www.usaa.com/inet/pages/security_token_logon_options
-
-    - name: UW Credit Union
-      url: https://www.uwcu.org/
-      img: uwcu.png
-      tfa: Yes
-      sms: Yes
-      software: Yes
-      phone: Yes
-      doc: https://www.uwcu.org/OnlineBanking/OnlineSecurity/VerifyU.aspx
-
-    - name: VyStar Credit Union
-      url: https://vystarcu.org/
-      twitter: OfficialVyStar
-      img: vystar.png
-      tfa: No
-
-    - name: Wells Fargo
-      url: https://www.wellsfargo.com/
-      img: wellsfargo.png
-      tfa: Yes
-      sms: Yes
-      phone: Yes
-      hardware: Yes
-      doc: https://www.wellsfargo.com/privacy-security/advanced-access
-
-    - name: Wescom Credit Union
-      url: https://www.wescom.org/
-      twitter: Liz_Wescom
-      img: wescom.png
-      tfa: No
-
-    - name: Zag Bank
-      url: https://www.zagbank.ca/
-      twitter: zagbank
-      img: zagbank.png
-      tfa: No
-
-    - name: ZenBanx
-      url: http://zenbanx.ca/
-      twitter: ZenBanx
-      img: zenbanx.png
-      tfa: No
-
-    - name: Zurich Cantonal Bank (ZKB)
-      url: https://www.zkb.ch/
-      img:  zkb.png
-      tfa: Yes
-      sms: Yes
-      email: Yes
-      phone: Yes
-      software: Yes
-      hardware: Yes
-      doc: https://www.zkb.ch/de/pr/pk/efinance/ebanking/login-verfahren.html
-
+      lang: fr
+
+    - name: Deutsche Kreditbank (DKB)
+      url: https://www.dkb.de/
+      img: dkb.png
+      tfa: No
+      facebook: Deutsche.Kreditbank
+      lang: de
+
+    - name: Discover
+      url: https://www.discover.com/
+      img: discover.png
+      tfa: no
+      twitter: Discover
+
+    - name: DNB
+      url: https://www.dnb.no/
+      img: dnb.png
+      tfa: Yes
+      hardware: Yes
+      software: Yes
+      doc: https://www.dnb.no/privat/kundeservice/innlogging-nettbank.html
+
+    - name: everbank.com
+      url: https://everbank.com
+      img: everbank.png
+      tfa: Yes
+      phone: Yes
+      sms: Yes
+
+    - name: Fairwinds
+      url: https://www.fairwinds.org
+      img: fairwinds.png
+      tfa: Yes
+      sms: Yes
+      email: Yes
+      phone: Yes
+      doc: "https://www.fairwinds.org/inside/privacy/security_we.asp#multifactor-auth"
+
+    - name: Fifth Third Bank
+      url: https://www.53.com/
+      twitter: fifththird
+      img: fifththird.png
+      tfa: No
+
+    - name: Final
+      url: https://www.getFinal.com/

```

Comment:
since domains are case insensitive and google return a no `www` URL, please correct `url` to `https://getfinal.com`